hass_nabucasa/voice.py

```diff
@@ -48,757 +48,9 @@ class AudioOutput(str, Enum):
     RAW = "raw"
 
 
-# The first entry for each language is the default voice
+# backwards compat
```

Comment:
I've kept this variable for backwards compatibility, to prevent requiring code changes when updating the library in Home Assistant. Once HA uses the new data, will drop it.